javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -110,7 +112,9 @@ public static ResettableClassFileTransformer installBytebuddyAgent(
 
     setBootstrapPackages(config);
 
-    runBeforeAgentListeners(agentListeners, config);
+    AutoConfiguredOpenTelemetrySdk autoConfiguredSdk = installOpenTelemetrySdk(config);
```

Comment:
`installOpenTelemetrySdk(Config)` is `@Nullable` because config may dictate a noop `OpenTelemetry`. But how is a noop `OpenTelemetry` different than just deactivating the agent altogether via `otel.javaagent.enabled=false`?

Reply:
see #3053 for history / purpose of noop